spec/Section 7 -- Response.md

```diff
@@ -244,6 +264,174 @@ discouraged.
 }
 ```
 
+### Path
+
+A `path` field allows for the association to a particular field in a GraphQL
+result. This field should be a list of path segments starting at the root of the
+response and ending with the field to be associated with. Path segments that
+represent fields should be strings, and path segments that represent list
+indices should be 0-indexed integers. If the path is associated to an aliased
+field, the path should use the aliased name, since it represents a path in the
+response, not in the request.
+
+When the `path` field is present on an "Error result", it indicates the response
+field which experienced the error.
+
```

Comment:
Should we extract this and line 136/137 into a separate PR to merge sooner?

Reply:
Opened https://github.com/graphql/graphql-spec/pull/1129